cas-server-integration-ehcache/src/main/java/org/jasig/cas/ticket/registry/EhCacheTicketRegistry.java

```diff
@@ -50,7 +51,7 @@
  * @author Andrew Tillinghast
  * @since 3.5
  */
-public final class EhCacheTicketRegistry extends AbstractDistributedTicketRegistry implements InitializingBean {
+public final class EhCacheTicketRegistry extends AbstractCrypticTicketRegistry implements InitializingBean {

```

Comment:
Also, bear in mind that if you are just running with the default ticket registry which is perfectly sufficient for single-node CAS deployment for smaller deployments, you do not need to enable encryption, which furthermore complicates your configuration if encryption support is enabled as a wrapper.